setup.cfg

```diff
@@ -160,5 +160,8 @@ timeout = 80
 filterwarnings =
     ignore::pytest.PytestWarning
 
+markers =
```

Comment:
how to use this?

Reply:
The `conftest.py` has an `allow_on_cloud` that can be used, see example in `test_group.py`